running-coreos/cloud-providers/vultr/index.md

```diff
@@ -11,10 +11,28 @@ weight: 10
 These instructions will walk you through running a single CoreOS node. This guide assumes:
 
 * You have an account at [Vultr.com](https://www.vultr.com).
-* The location of your iPXE script (referenced later in the guide) is located at ```http://example.com/script.txt```
 * You have a public + private key combination generated. Here's a helpful guide if you need to generate these keys: [How to set up SSH keys](https://help.github.com/articles/generating-ssh-keys).
 
-The simplest option to boot up CoreOS is to load a script that contains the series of commands you'd otherwise need to manually type at the command line. This script needs to be publicly accessible (host this file on your own server). Save this script as a text file (.txt extension).
+The simplest option to boot up CoreOS is to select the "CoreOS Stable" operating system from Vultr's default offerings. However, most deployments require a custom `cloud-config`, which can only be achieved in Vultr with an iPXE script. The remainder of this article describes this process.
+
+## Cloud-Config
+
+First, you'll need to make two files available at public URL's -- your `cloud-config` file and a shell script which you need to create:
+
+```sh
+URL="http://example.com/cloud-config.yaml"
+
+cat << EOF > ./cloud-config-bootstrap.sh\n
```

Comment:
Thanks. Just switching to your HEREDOC format worked. Maybe it was just a bash-version incompatibility because the previous version worked fine in my local bash.